examples/combobox/aria1.1pattern/listbox-combo.html

```diff
@@ -274,7 +274,7 @@ <h3 id="kbd_label_listbox">Listbox Popup</h3>
               </ul>
             </td>
           </tr>
-          <tr>
+          <tr data-test-id="key-down-arrow">
```

Comment:
This is same ID as line 185, but won't the tests be different?

Reply:
you are right, and it lead to me to not test this row of this table. Changing this to: `listbox-key-down-arrow` and the other to `textbox-key-down-arrow`